### PR TITLE
FIDES-1554: Update GPP Package

### PR DESCRIPTION
**Pull request by tvandort**
- **Update GPP package.**
- **Update tests.**

Closes FIDES-1554

### Description Of Changes

Update to newest version of GPP package.

### Code Changes

* Update tests.

### Pre-Merge Checklist

* [ ] Issue requirements met
* [ ] All CI pipelines succeeded
* [ ] `CHANGELOG.md` updated
* Followup issues:
  * [ ] Followup issues created (include link)
  * [ ] No followup issues
* Database migrations:
  * [ ] Ensure that your downrev is up to date with the latest revision on `main`
  * [ ] Ensure that your `downgrade()` migration is correct and works
    * [ ] If a downgrade migration is not possible for this change, please call this out in the PR description!
  * [ ] No migrations
* Documentation:
  * [ ] Documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] Documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
  * [ ] If there are any new client scopes created as part of the pull request, remember to update public-facing documentation that references our scope registry
  * [ ] No documentation updates required

### CODE REVIEW

File: clients/fides-js/__tests__/lib/gpp/stub.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ describe("makeStub", () => {
         expect(success).toBe(true);
         expect(data).toBe(false);
       },
-      "tcfcav1",
```
[the remark this answers is not in the document]
Comment (tvandort) [reply clients/fides-js/__tests__/lib/gpp/stub.ts:78]: Hm that would seem to be correct.

File: clients/fides-js/rollup.config.mjs [hunk 2]
```diff
@@ -109,7 +109,7 @@ const SCRIPTS = [
   {
     name: `${NAME}-ext-gpp`,
     gzipWarnSizeKb: 10,
-    gzipErrorSizeKb: 15,
+    gzipErrorSizeKb: 20,
```
Comment (gilluminate) [inline clients/fides-js/rollup.config.mjs:112]: @NevilleS FYI. The package itself added this weight.

File: clients/fides-js/__tests__/lib/gpp/stub.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tvandort) [reply clients/fides-js/__tests__/lib/gpp/stub.ts]: Changed this back.

**Review by gilluminate — COMMENTED** [review]
Code looks good. Fairly straight forward. Pulling down to test locally briefly before approving.

**Review by gilluminate — APPROVED** [review]
Tested locally with appropriate MSPA settings and working as expected.

**Pull request merged**